layout/theme.liquid

```diff
@@ -123,7 +123,7 @@
           </div>
         </div>
         {% else %}
-          {% unless current_user.is_anonymous? %}
+          {% if current_user.is_wholesale? %}

```

Comment:
@etagwerker @cecilia-ombulabs I'm thinking `if current_user.is_wholesale? || current_user.is_customer?` should do it. The problem with this code is that if the user sends a contact message, the `current_user` now has an email associated to it, which no longer makes him anonymous, and displays the wholesale bar because they aren't anonymous anymore.